src/backend/app/db/models.py

```diff
@@ -13,3 +13,4 @@ class AnnotationHexagon(Base):
     annotation_hexagon_id = Column(Integer, primary_key=True, index=True)
     annotation_id = Column(Integer, ForeignKey("annotation.annotation_id"))
     hex_id = Column(String)
+    type = Column(String)
```

Comment:
Per team discussion at standup:
```suggestion
    hex_type = Column(String)
```

Reply:
Changed to hex_type in the database proposal doc too!